### ISSUE TITLE
Tweak searchLabel field analysers

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/next-es-interface#740]
At the moment we're using the default settings for word separators when generating the `searchLabel` for each metadata `prefLabel` label field. This is giving us problems for some users attempting to use typeahead search, for example:

What the user types | What this API generates | What Elasticsearch has generated
--------------------|-------------------------|---------------------------------
Co-operative        | cooperat                | co operativ
T-mobile            | tmobile                 | mobile

I think we have a few options:

- Allow ngrams of single character length (`"min_gram": 1`)
- Use a custom [word delimeter token filter](https://www.elastic.co/guide/en/elasticsearch/reference/1.5/analysis-word-delimiter-tokenfilter.html) to concatenate dashes

Either way requires creation of a new index.

**Comment by bjfletcher** [comment]
@i-like-robots can you clarify on:

> though that may cause problems for users searching with space separated terms.

example?

**Comment by i-like-robots** [comment]
If we concatenated `Co-operative` to `cooperative` then that would not match if user was looking for the term `co op`. This is because we currently generate _edge_ ngrams between 2-8 chars only, E.G. given "Co-operative" this is what Elasticsearch generates:

```
co
coo
coop
coope
cooper
coopera
cooperat
```

(you can test our ngram generation configuration [here](http://next-elastic.ft.com/content/_analyze?field=metadata.searchLabel&text=co-operative))

I had initially thought when writing the issue that to get around this problem we could switch to _non-edge_ ngram generation, that would mean ES would create something like this:

```
co
coo
coop
coope
cooper
coopera
cooperat
ooperata
operatav
perative
... etc
```

However, I know remember this type of break-down is quite useless. In this case it would match if somebody was typing in "rat" for instance... and I think it's fair to assume users don't start typing the middle or end of a word!

In summary... I think option 2 is actually not feasible anyway!

**Comment by i-like-robots** [comment]
Fixed in #763
